agimus_controller/tests/test_ocp_param_base.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import numpy as np
+
+<<<<<<< Updated upstream
+from agimus_controller.agimus_controller.ocp_param_base import OCPParamsCrocoBase
+from agimus_controller.agimus_controller.trajectory import (
+    WeightedTrajectoryPoint,
+    TrajectoryPoint,
+    TrajectoryPointWeights,
+)
+
+=======
+from agimus_controller.ocp_param_base import OCPParamsCrocoBase
+from agimus_controller.trajectory import WeightedTrajectoryPoint, TrajectoryPoint, TrajectoryPointWeights
+>>>>>>> Stashed changes
+
+class TestOCPParamsCrocoBase(unittest.TestCase):
+    """Test the OCPParamsCrocoBase class."""
+
+    def __init__(self, methodName="runTest"):
+        super().__init__(methodName)
+
+    def test_initialization(self):
+        dt = np.float64(0.01)
+        T = 100
+        solver_iters = 50
```

Comment:
You don't have to cast those and you can pass them in the `OCPParamsCrocoBase` directly